examples/misc/lib/language_tour/misc/null_aware_spread_operator_in_collection_a.dart

```diff
@@ -0,0 +1,9 @@
+// void main() {
+//   // #docregion code_sample
+//   var a = null;
+//   var b = [1, 2, null];
```

Comment:
Nit: It might be a little clearer as `1, null, 2` and likewise below. That way it's little more obvious that the `null` in the resulting list comes from the middle of this one.

Reply:
Updated! I used [1, null, 3]